genie-web/src/test/resources/application-ci.yml

```diff
@@ -30,6 +30,10 @@ genie:
   health:
     maxCpuLoadPercent: 100
   jobs:
+    clusters:
```

Comment:
`application.yml` above does not rely on default property values, whereas this and the next conf do. Is this intentional.